PixelUnion/Tests/AllSniffs.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace PixelUnion\Tests;
+
+use PHP_CodeSniffer\Autoload;
+use PHPUnit\TextUI\TestRunner;
+use PHPUnit\Framework\TestSuite;
+
+class AllSniffs
+{
+    /**
+     * Prepare the test runner.
+     *
+     * @return void
+     */
+    public static function main()
+    {
+        TestRunner::run(self::suite());
+    }
+
+    /**
+     * Add all sniff unit tests into a test suite.
+     *
+     * Sniff unit tests are found by recursing through the 'Tests' directory
+     *
+     * @return \PHPUnit\Framework\TestSuite
+     */
+    public static function suite()
+    {
+        $suite = new TestSuite('Pixel Union CodeSniffing Standard');
+
+        $testsDir = __DIR__;
+
+        $di = new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($testsDir));
+        foreach ($di as $file) {
+            $parts = explode('.', $file);
+
+            // Tests must have the extension 'php'.
+            $ext   = array_pop($parts);
+            if ($ext !== 'php') {
+                continue;
+            }
+
+            // Tests must end in 'Test'.
+            $fileName = array_pop($parts);
+            if (substr_compare($fileName, 'Test', -strlen('Test'))) {
```

Comment:
- This seems to need a `!`. 
- Edge cases for files with no extensions or file names starting with `.` will cause exception.